common/culture/traditions/wc_societal_traditions.txt

```diff
@@ -0,0 +1,374 @@
+		if = {
+			limit = {
+				scope:character.faith = {
+					has_doctrine = doctrine_pluralism_fundamentalist
+				}
+			}
+			multiply = 1.15
+		}
+	}
+}
+
+tradition_bandit_culture = {
+	category = societal
+
+	layers = {
+		0 = martial
+		1 = western
+		4 = bandits.dds
+	}
+	
+	is_shown = {
+		NOT = { has_cultural_tradition = tradition_subservient_soldiers }
+	}
+	can_pick = {
+		culture_not_pacifistic_trigger = yes
+		scope:character = {
+			has_government = tribal_government
+		}
+		custom_tooltip = {
+			text = cannot_have_tradition_fp1_coastal_warriors
+			NOT = { any_in_list = { list = traits this = flag:tradition_fp1_coastal_warriors } }
+		}
+	}
+	
+	parameters = {
+		wounds_and_scars_give_bonuses = yes
+		weak_traits_looked_down_upon = yes
+		rowdy_trait_more_common = yes
+		craven_and_content_traits_looked_down_upon = yes
+	}
+	combat_modifier = {
+		supply_duration = .05
+	}
+	character_modifier = {
+		mercenary_hire_cost_mult = -0.5
+	}
+	cost = {
+		prestige = {
+			add = {
+				value = tradition_base_cost
+				desc = BASE
+				format = "BASE_VALUE_FORMAT"
+			}
+			if = {
+				limit = {
+					NOR = {
+						any_in_list = { list = traits this = flag:ethos_bellicose }
+						any_in_list = { list = traits this = flag:ethos_spiritual }
+					}
+				}
+				add = {
+					value = tradition_incompatible_ethos_penalty
+					desc = not_bellicose_or_spiritual_desc
+				}
+			}
+			if = {
+				limit = {
+					trigger_if = {
+						limit = {
+							scope:character = {
+								is_ai = no
+							}
+						}
+						NOT = {
+							any_ruler = {
+								count >= 3
+								culture = prev
+								primary_title.tier >= tier_county
+								OR = {	
+									has_trait = strong
+									has_trait = physique_good
+								}
+							}
+						}
+					}
+					trigger_else = {
+						always = no
+					}
+				}
+				add = {
+					value = tradition_unfulfilled_criteria_penalty
+					desc = strong_rulers_desc
+				}
+			}
+			
+			multiply = tradition_replacement_cost_if_relevant
+		}
+	}
+	ai_will_do = {
+		value = 100
+		if = {
+			limit = {
+				NOT = {
+					any_in_list = { list = traits this = flag:ethos_bellicose }
+				}
+			}
+			multiply = 0
+		}
+	}
+}
+
+tradition_time_is_money = {
+	category = societal
+
+	layers = {
+		0 = steward
+		1 = mena
+		4 = time_is_money.dds
+	}
+	
+	parameters = {
+		development_gain_on_building_complete = yes
+	}
+	character_modifier = {
+		monthly_income_mult = 0.05
+		vassal_tax_contribution_mult = 0.1
+	}
+	
+	cost = {
+		prestige = {
+			add = {
+				value = tradition_base_cost
+				desc = BASE
+				format = "BASE_VALUE_FORMAT"
+			}
+			if = {
+				limit = {
+					NOR = {
+						any_in_list = { list = traits this = flag:ethos_bellicose }
+						any_in_list = { list = traits this = flag:ethos_stoic }
+						any_in_list = { list = traits this = flag:ethos_communal }
+					}
+				}
+				add = {
+					value = tradition_incompatible_ethos_penalty
+					desc = not_bellicose_stoic_or_communal_desc
+				}
+			}
```

Comment:
Weird penalties. I think only `ethos_bureaucratic` should have it without penalty.